index.html

```diff
@@ -1,133 +1,111 @@
-            </div>
-            <div class="speacker">
-                <div>
-                <img class="speackerimg" src="./img/speackertwo.jpg" alt="">
-                </div>
-                <div>
-                    <h2>Mark Zuckerberg</h2>
-                    <h4>Co-founder, Chairperson and CEO Facebook</h4>
-                    <P>Mark Elliot Zuckerberg is a media magnate, internet entrepreneur, <br> 
-                        and philanthropist. He is known for co-founding Facebook</P>
-                </div>
-            </div>
-            <div class="speacker">
-                <div>
-                <img class="speackerimg" src="./img/speackerthree.jpg" alt="">
-                </div>
-                <div>
-                    <h2>Amina Mohammed</h2>
-                    <h4>Deputy Secretary-General United Nations</h4>
-                    <p>Ms. Amina J. Mohammed is the Deputy Secretary-General of the United Nations <br> 
-                        and Chair of the United Nations Sustainable Development Group.</p>
-                </div>
-            </div>
-            <div class="speacker">
-                <div>
-                <img class="speackerimg" src="./img/speackerfour.jpg" alt="">
-                </div>
-                <div>
-                    <h2>Cher Wang</h2>
-                    <h4>Co-founder and Chairperson, HTC corporation</h4>
-                    <p>Cher Wang is an entrepreneur and philanthropist. As co-founder and chairperson <br>
-                         (since 2007) of HTC Corporation and integrated chipset maker </p>
-                </div>
-            </div>
-            <div class="speacker">
-                <div> 
-                <img class="speackerimg" src="./img/speackerfive.jpg" alt="">
-                </div>
-                <div>
-                    <h2>Reed Hastings</h2>
-                    <h4>CEO Netflix</h4>
-                    <p>Is a businessman. He is the co-founder, chairman, and co-chief executive officer (CEO) of Netflix.</p>
-                </div>
+        <div class="container">
+            <div>
+                <h2 class="featured">Featured Speackers</h2>
             </div>
+            <div class="speackersContainer" id="speackersContainer"></div>
         </div>
-
-      </div>
     </section>
-
-    <footer></footer>
+    <section class="partnersection">
+        <h2 class="partnertext">Partners</h2>
+        <div class="partners">
+            <img class="partnericon" src="./img/sponsor1.jpg" alt="">
+            <img class="partnericon" src="./img/logospnsor2.png" alt="">
+            <img class="partnericon" src="./img/logosponsor4.png" alt="">
+            <img class="partnericon" src="./img/logosponsor5.png" alt="">
+            <img class="partnericon" src="./img/logosponsor6.png" alt="">
+        </div>
+    </section>
+    <footer>
+        <img src="./img/logo3.png" alt="">
+        <p>GSMA logoLegal© Copyright 2021 GSM Association.</p>
+    </footer>
```

Comment:
- [ ] In the original design I can see a footer and a partner's section but on yours, I can see neither the footer nor the partner's section. Please kindly fix that

Reply:
I have a footer and a partner's section, I guess that you checked previous branches. Can you please check again with the current branch?